install/amazon_rds/01_configure_rds_instance.mdx

```diff
@@ -17,10 +17,10 @@ Connect to your database as an RDS superuser (usually the credentials you create
 
 Run the following SQL commands to enable the extension, and make sure it was installed correctly:
 
-```
-CREATE EXTENSION IF NOT EXISTS pg_stat_statements;
-SELECT * FROM pg_stat_statements LIMIT 1;
-```
+<CodeBlock language="sql">
+{`CREATE EXTENSION IF NOT EXISTS pg_stat_statements;
+SELECT * FROM pg_stat_statements LIMIT 1;`}
+</CodeBlock>
```

Comment:
If, instead, this text looked like this:
```
<CodeBlock language="sql">
{`CREATE EXTENSION IF NOT EXISTS pg_stat_statements;

SELECT * FROM pg_stat_statements LIMIT 1;`}
</CodeBlock>
```
it would throw an error that the first line is missing an expected `\`}` token.

However, this renders OK visually (and through copy button):
```<CodeBlock language="sql">
{`CREATE EXTENSION IF NOT EXISTS pg_stat_statements;\n
SELECT * FROM pg_stat_statements LIMIT 1;`}
</CodeBlock>